`python39Packages.sentry-sdk` build failure on x86_64-linux as of `891d0226`

## Issue description
Build of `python39Packages.sentry-sdk` failed on x86_64-linux as of 891d0226de6e281422a8d0d99814925fd7aed8d0. This is currently breaking python3Packages.wandb.

```
error: builder for '/nix/store/j4lfky79xqjiabprrqzba2d8mjm6rhf4-python3.9-sentry-sdk-1.5.8.drv' failed with exit code 1;
       last 10 log lines:
       > ERROR tests/integrations/flask/test_flask.py::test_errors_not_reported_twice[integrations0]
       > ERROR tests/integrations/flask/test_flask.py::test_errors_not_reported_twice[integrations1]
       > ERROR tests/integrations/flask/test_flask.py::test_logging - werkzeug.excepti...
       > ERROR tests/integrations/flask/test_flask.py::test_500 - werkzeug.exceptions....
       > ERROR tests/integrations/flask/test_flask.py::test_error_in_errorhandler - we...
       > ERROR tests/integrations/flask/test_flask.py::test_tracing_success - werkzeug...
       > ERROR tests/integrations/flask/test_flask.py::test_tracing_error - werkzeug.e...
       > ERROR tests/integrations/flask/test_flask.py::test_class_based_views - werkze...
       > ERROR tests/integrations/flask/test_flask.py::test_sentry_trace_context - wer...
       > = 37 failed, 563 passed, 32 skipped, 18 deselected, 2 xfailed, 17 warnings, 38 errors in 75.70s (0:01:15) =
       For full logs, run 'nix log /nix/store/j4lfky79xqjiabprrqzba2d8mjm6rhf4-python3.9-sentry-sdk-1.5.8.drv'.
error: 1 dependencies of derivation '/nix/store/ygwhmibr8flr7xxxnif44w2wbi8i9fvq-python3.9-wandb-0.12.14.drv' failed to build
```

[full build log](https://github.com/samuela/nixpkgs-upkeep/actions/runs/2181534947)

python3Packages.wandb maintainers cc: @samuela
python39Packages.sentry-sdk maintainers cc: @fabaff @gebner
Other cc: n/a

### Steps to reproduce
1. Checkout nixpkgs at commit 891d0226de6e281422a8d0d99814925fd7aed8d0
2. Run `nix-build -A python39Packages.sentry-sdk`

## Technical details
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.13.0-1021-azure, Ubuntu, 20.04.4 LTS (Focal Fossa), nobuild`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.7.0`
 - nixpkgs: `/nix/store/r902mrj2iwsjhy8gs6y0rbr9nada0z4z-source`
```

## Misc.
This issue was automatically generated by [nixpkgs-upkeep](https://github.com/samuela/nixpkgs-upkeep).
- [CI workflow](https://github.com/samuela/nixpkgs-upkeep/actions/runs/2181534947) that created this issue.
- Internal tag: 3211800932645290548

I also tried bisecting. Due to the massive rebuilds, I canceled after 8 hours and did a bisect with `first-parent`.
The bad commit is 95af2245a32f8e1310ad4e3bf50b76d86ddbbc0a from #168737, which is a merge from staging with changes to python and others:

<details>

```
95af2245a32f8e1310ad4e3bf50b76d86ddbbc0a is the first bad commit
commit 95af2245a32f8e1310ad4e3bf50b76d86ddbbc0a
Merge: 212dde2609b 70d726c2a51
Author: Martin Weinelt <mweinelt@users.noreply.github.com>
Date:   Sun Apr 17 20:38:16 2022 +0200

    Merge pull request #168737 from NixOS/staging-next

 doc/languages-frameworks/gnome.section.md          |  18 +-
 .../from_md/release-notes/rl-2205.section.xml      |  19 +
 nixos/doc/manual/release-notes/rl-2205.section.md  |   4 +
 nixos/modules/misc/locate.nix                      |   2 +-
 nixos/modules/services/misc/etebase-server.nix     |   2 +-
 nixos/modules/services/misc/gitlab.nix             |   4 +-
 pkgs/applications/audio/hivelytracker/default.nix  |   2 +-
 .../audio/hqplayer-desktop/default.nix             |   1 +
 pkgs/applications/audio/libopenmpt/default.nix     |  42 --
 .../audio/losslessaudiochecker/default.nix         |   2 +-
 pkgs/applications/audio/lyrebird/default.nix       |   2 +-
 pkgs/applications/audio/puddletag/default.nix      |   1 +
 .../audio/pulseaudio-modules-bt/default.nix        |  91 ---
 .../pulseaudio-modules-bt/fix-install-path.patch   |  11 -
 .../applications/audio/soundwireserver/default.nix |   1 +
 .../blockchains/wasabibackend/default.nix          |   2 +-
 pkgs/applications/editors/pinegrow/default.nix     |   2 +-
 pkgs/applications/editors/vim/common.nix           |   4 +-
 pkgs/applications/graphics/awesomebump/default.nix |   2 +-
 pkgs/applications/graphics/cq-editor/default.nix   |   2 +-
 pkgs/applications/graphics/foxotron/default.nix    |   2 +-
 pkgs/applications/graphics/pixeluvo/default.nix    |   1 +
 .../graphics/unigine-valley/default.nix            |   2 +-
 pkgs/applications/misc/buku/default.nix            |   3 +-
 pkgs/applications/misc/foxitreader/default.nix     |   1 +
 pkgs/applications/misc/multibootusb/default.nix    |   2 +-
 pkgs/applications/misc/octoprint/default.nix       |   4 +-
 pkgs/applications/misc/tvbrowser/bin.nix           |   2 +-
 pkgs/applications/misc/upwork/default.nix          |   1 +
 pkgs/applications/misc/xteddy/default.nix          |   2 +-
 .../networking/browsers/firefox/common.nix         |   5 +-
 .../cluster/ssm-session-manager-plugin/default.nix |   7 +-
 .../instant-messengers/ripcord/default.nix         |   2 +-
 .../instant-messengers/torchat/default.nix         |   2 +-
 .../instant-messengers/zoom-us/default.nix         |   2 +-
 .../networking/mailreaders/mailspring/default.nix  |   2 +-
 .../mailreaders/thunderbird-bin/default.nix        |   4 +-
 .../networking/p2p/tribler/default.nix             |   4 +-
 .../networking/remote/rustdesk/default.nix         |   2 +-
 pkgs/applications/networking/seahub/default.nix    |  17 +-
 pkgs/applications/networking/tetrd/default.nix     |   2 +-
 pkgs/applications/office/fava/default.nix          |  18 +-
 pkgs/applications/office/jameica/default.nix       |   2 +-
 pkgs/applications/science/biology/cmtk/default.nix |   5 +-
 .../science/logic/fast-downward/default.nix        |  31 +-
 .../applications/science/logic/tlaplus/toolbox.nix |   2 +-
 pkgs/applications/science/misc/fityk/default.nix   |   4 +
 pkgs/applications/science/misc/netlogo/default.nix |   2 +-
 .../science/robotics/sumorobot-manager/default.nix |   2 +-
 .../terminal-emulators/syncterm/default.nix        |   2 +
 .../git-and-tools/git/default.nix                  |  22 +-
 pkgs/applications/video/epgstation/default.nix     |   2 +-
 pkgs/applications/video/mirakurun/default.nix      |   4 +-
 .../window-managers/afterstep/default.nix          |  14 +-
 pkgs/build-support/setup-hooks/make-wrapper.sh     |   5 +
 .../setup-hooks/wrap-gapps-hook/wrap-gapps-hook.sh |   4 -
 pkgs/build-support/trivial-builders.nix            |  10 +-
 pkgs/data/misc/cacert/default.nix                  |   4 +-
 pkgs/data/misc/cacert/update.sh                    |  17 +-
 pkgs/data/misc/dns-root-data/default.nix           |   3 +-
 pkgs/data/misc/freepats/default.nix                |   5 +-
 pkgs/desktops/cdesktopenv/default.nix              |  13 +
 pkgs/desktops/gnome/core/mutter/3.38/default.nix   |   4 +-
 pkgs/desktops/gnome/core/mutter/default.nix        |   4 +-
 pkgs/development/compilers/dmd/binary.nix          |   5 +-
 pkgs/development/compilers/go/1.16.nix             |   3 +-
 pkgs/development/compilers/go/1.17.nix             |   7 +-
 pkgs/development/compilers/go/1.18.nix             |   3 +-
 pkgs/development/compilers/inklecate/default.nix   |   2 +-
 pkgs/development/compilers/ldc/binary.nix          |   5 +-
 .../development/compilers/llvm/10/llvm/default.nix |   6 +
 .../development/compilers/llvm/11/llvm/default.nix |   6 +
 .../development/compilers/llvm/12/llvm/default.nix |   6 +
 .../development/compilers/llvm/13/llvm/default.nix |   6 +
 .../development/compilers/llvm/14/llvm/default.nix |   6 +
 pkgs/development/compilers/llvm/5/llvm/default.nix |   6 +
 pkgs/development/compilers/llvm/6/llvm/default.nix |   6 +
 pkgs/development/compilers/llvm/7/llvm/default.nix |   6 +
 pkgs/development/compilers/llvm/8/llvm/default.nix |   6 +
 pkgs/development/compilers/llvm/9/llvm/default.nix |   6 +
 .../compilers/llvm/git/llvm/default.nix            |   6 +
 pkgs/development/compilers/mrustc/default.nix      |   1 +
 pkgs/development/compilers/rust/1_59.nix           |  64 --
 pkgs/development/compilers/rust/1_60.nix           |  64 ++
 .../compilers/rust/rustfmt-fix-self-tests.patch    |  13 -
 pkgs/development/compilers/rust/rustfmt.nix        |   3 -
 pkgs/development/compilers/zulu/8.nix              |  12 +-
 pkgs/development/compilers/zulu/default.nix        |   8 +-
 pkgs/development/go-modules/generic/default.nix    |   3 +-
 pkgs/development/go-packages/generic/default.nix   |   3 +-
 pkgs/development/interpreters/gnu-apl/default.nix  |   4 +-
 pkgs/development/interpreters/perl/default.nix     |  23 +-
 .../interpreters/python/cpython/default.nix        |   2 +-
 pkgs/development/interpreters/python/default.nix   |   8 +-
 pkgs/development/interpreters/ruby/default.nix     |  56 +-
 pkgs/development/interpreters/ruby/patchsets.nix   |   6 +-
 .../ruby/rbinstall-new-rubygems-compat.patch       |  87 +++
 pkgs/development/libraries/SDL2/default.nix        |   6 +-
 .../libraries/audio/libopenmpt/default.nix         |  66 ++
 .../libraries/audio/libopenmpt/update.sh           |  31 +
 pkgs/development/libraries/audio/lilv/default.nix  |   1 +
 pkgs/development/libraries/audio/lv2/default.nix   |   2 +
 .../development/libraries/audio/sratom/default.nix |   1 +
 .../libraries/aws-c-sdkutils/default.nix           |   4 +-
 pkgs/development/libraries/belle-sip/default.nix   |   3 +-
 .../development/libraries/clucene-core/default.nix |   4 +
 .../development/libraries/cpp-ipfs-api/default.nix |   4 +
 pkgs/development/libraries/cutelyst/default.nix    |  17 +-
 pkgs/development/libraries/flatcc/default.nix      |   5 +
 pkgs/development/libraries/freetype/default.nix    |   4 +-
 pkgs/development/libraries/glib/default.nix        |   6 +
 .../glib/elementary-terminal-support.patch         |  16 +
 .../libraries/glib/gnome-console-support.patch     |  55 ++
 pkgs/development/libraries/glm/default.nix         |   5 +
 pkgs/development/libraries/glog/default.nix        |  19 +-
 pkgs/development/libraries/grpc/default.nix        |   4 +-
 .../gsettings-desktop-schemas/default.nix          |  12 +-
 pkgs/development/libraries/hwloc/default.nix       |   4 +-
 pkgs/development/libraries/kde-frameworks/fetch.sh |   2 +-
 pkgs/development/libraries/kde-frameworks/srcs.nix | 664 ++++++++++-----------
 pkgs/development/libraries/libarchive/default.nix  |  93 ++-
 pkgs/development/libraries/libargon2/default.nix   |   4 +-
 pkgs/development/libraries/libavif/default.nix     |   4 +-
 pkgs/development/libraries/libepoxy/default.nix    |   5 +-
 pkgs/development/libraries/libevdev/default.nix    |   4 +-
 pkgs/development/libraries/libgcrypt/default.nix   |   4 +-
 pkgs/development/libraries/libjson/default.nix     |   1 +
 pkgs/development/libraries/libmnl/default.nix      |   4 +-
 .../Use-stdlib-uint-instead-of-u_int.patch         | 499 ----------------
 .../development/libraries/libnfnetlink/default.nix |   8 +-
 pkgs/development/libraries/libotr/default.nix      |   2 +
 .../libraries/libotr/fix-regtest-client.patch      |  11 +
 pkgs/development/libraries/libsoup/3.x.nix         |   4 +-
 .../development/libraries/libunistring/default.nix |   4 +-
 pkgs/development/libraries/libunwind/default.nix   |  10 +-
 pkgs/development/libraries/libwacom/default.nix    |   4 +-
 pkgs/development/libraries/log4shib/default.nix    |   2 +
 .../libraries/mapbox-gl-native/default.nix         |   5 +
 .../libraries/mediastreamer/default.nix            |   4 +-
 pkgs/development/libraries/mesa/default.nix        |   6 +-
 pkgs/development/libraries/mpich/default.nix       |   3 +
 pkgs/development/libraries/mvapich/default.nix     |   1 +
 .../libraries/netcdf-fortran/default.nix           |   3 +
 pkgs/development/libraries/netcdf/default.nix      |  22 +-
 pkgs/development/libraries/nss/default.nix         | 199 ------
 pkgs/development/libraries/nss/esr.nix             |   4 +
 pkgs/development/libraries/nss/generic.nix         | 193 ++++++
 pkgs/development/libraries/nss/latest.nix          |  10 +
 pkgs/development/libraries/openvino/default.nix    |   2 +
 pkgs/development/libraries/pcre/default.nix        |   2 +-
 pkgs/development/libraries/pcre2/default.nix       |   4 +-
 .../libraries/physics/qcdnum/default.nix           |   4 +
 pkgs/development/libraries/pmdk/default.nix        |   6 +-
 pkgs/development/libraries/polkit-qt-1/default.nix |   4 +-
 pkgs/development/libraries/qrupdate/default.nix    |  14 +-
 pkgs/development/libraries/qt-5/5.12/default.nix   |   8 +-
 pkgs/development/libraries/qt-5/5.14/default.nix   |   9 +-
 .../qt-5/5.14/qtwayland-libdrm-build.patch         |  56 --
 pkgs/development/libraries/qt-5/5.15/default.nix   |   8 +-
 .../libraries/qt-5/5.15/srcs-generated.json        | 126 ++--
 pkgs/development/libraries/qt-5/modules/qtbase.nix |  14 +-
 .../libraries/qt-5/modules/qtwayland.nix           |   6 +
 pkgs/development/libraries/qt-5/qtModule.nix       |   2 +-
 pkgs/development/libraries/re2/default.nix         |   4 +-
 pkgs/development/libraries/rlottie/default.nix     |  17 +-
 pkgs/development/libraries/smesh/default.nix       |   2 +
 pkgs/development/libraries/talloc/default.nix      |   3 +-
 pkgs/development/libraries/tpm2-tss/default.nix    |   8 +-
 .../libraries/zlib/CVE-2018-25032-1.patch          | 346 +++++++++++
 .../libraries/zlib/CVE-2018-25032-2.patch          |  27 +
 pkgs/development/lua-modules/overrides.nix         |   3 +-
 pkgs/development/misc/msp430/mspds/binary.nix      |   3 +-
 pkgs/development/mobile/androidenv/build-tools.nix |   4 +-
 .../mobile/androidenv/platform-tools.nix           |   3 +-
 pkgs/development/python-modules/Mako/default.nix   |  35 +-
 pkgs/development/python-modules/aesara/default.nix |   6 +-
 pkgs/development/python-modules/afdko/default.nix  |  12 +-
 .../python-modules/aiobotocore/default.nix         |   4 +-
 pkgs/development/python-modules/aioftp/default.nix |   4 +-
 .../development/python-modules/alembic/default.nix |   8 +-
 pkgs/development/python-modules/amqp/default.nix   |   4 +-
 .../python-modules/ansible-lint/default.nix        |   4 -
 .../python-modules/ansible-runner/default.nix      |   5 +
 .../development/python-modules/apprise/default.nix |   4 +-
 .../python-modules/approvaltests/default.nix       |  42 +-
 pkgs/development/python-modules/apsw/default.nix   |   7 +-
 pkgs/development/python-modules/asana/default.nix  |  42 +-
 .../development/python-modules/asgiref/default.nix |  18 +-
 .../development/python-modules/astroid/default.nix |   8 +-
 .../astropy-extension-helpers/default.nix          |   5 +
 .../development/python-modules/astropy/default.nix |   4 +-
 .../python-modules/asttokens/default.nix           |   7 +-
 .../python-modules/async_generator/default.nix     |   2 -
 .../atlassian-python-api/default.nix               |   4 +-
 pkgs/development/python-modules/atom/default.nix   |  44 +-
 .../python-modules/aurorapy/default.nix            |   2 +
 .../python-modules/autobahn/default.nix            |   4 +-
 .../python-modules/azure-core/default.nix          |   8 +-
 .../python-modules/azure-mgmt-core/default.nix     |   2 +
 .../python-modules/azure-mgmt-msi/default.nix      |   4 +-
 pkgs/development/python-modules/bcrypt/default.nix |  41 +-
 .../python-modules/beautifultable/default.nix      |   7 +-
 pkgs/development/python-modules/behave/default.nix |   4 +-
 .../python-modules/bitarray/default.nix            |   4 +-
 .../python-modules/bitstruct/default.nix           |   4 +-
 pkgs/development/python-modules/black/default.nix  |   4 +-
 .../development/python-modules/blinker/default.nix |  11 +-
 pkgs/development/python-modules/boto3/default.nix  |   4 +-
 .../python-modules/botocore/default.nix            |   4 +-
 pkgs/development/python-modules/boxx/default.nix   |   4 +-
 pkgs/development/python-modules/build/default.nix  |   2 -
 .../python-modules/buildbot/default.nix            |   4 +-
 pkgs/development/python-modules/buildbot/pkg.nix   |   2 +-
 .../python-modules/buildbot/plugins.nix            |  10 +-
 .../development/python-modules/buildbot/worker.nix |  48 +-
 pkgs/development/python-modules/cattrs/default.nix |   3 -
 pkgs/development/python-modules/cffi/default.nix   |   9 +-
 .../development/python-modules/chalice/default.nix |   2 +-
 .../python-modules/cherrypy/default.nix            |   4 +-
 .../python-modules/ckcc-protocol/default.nix       |   4 +-
 .../development/python-modules/claripy/default.nix |   2 +
 .../python-modules/click-log/default.nix           |   4 +-
 pkgs/development/python-modules/click/default.nix  |  11 +-
 .../python-modules/clickhouse-driver/default.nix   |   2 -
 pkgs/development/python-modules/clize/default.nix  |  35 +-
 pkgs/development/python-modules/cppy/default.nix   |  19 +-
 .../python-modules/cryptography/default.nix        |  50 +-
 .../python-modules/cryptography/vectors.nix        |  15 +-
 pkgs/development/python-modules/dask/default.nix   |   2 -
 .../python-modules/datasette/default.nix           |  24 +-
 .../python-modules/datashader/default.nix          |   1 -
 .../python-modules/dateparser/default.nix          |   4 +-
 .../python-modules/debtcollector/default.nix       |   4 +-
 .../development/python-modules/debugpy/default.nix |   1 -
 .../python-modules/dictpath/default.nix            |   2 +
 .../python-modules/diskcache/default.nix           |   4 -
 .../python-modules/django-allauth/default.nix      |  36 +-
 pkgs/development/python-modules/django/3.nix       |   4 +-
 pkgs/development/python-modules/django/4.nix       |   4 +-
 .../python-modules/django_modelcluster/default.nix |  36 +-
 .../python-modules/dm-haiku/default.nix            |   1 -
 .../python-modules/dnspython/default.nix           |   4 +-
 .../python-modules/docutils/default.nix            |   4 +-
 pkgs/development/python-modules/dugong/default.nix |   5 +
 .../python-modules/entrypoint2/default.nix         |   4 +-
 .../ephemeral-port-reserve/default.nix             |  42 ++
 .../python-modules/exchangelib/default.nix         |   6 +-
 pkgs/development/python-modules/faker/default.nix  |   4 +-
 .../python-modules/fakeredis/default.nix           |   5 +
 pkgs/development/python-modules/falcon/default.nix |  79 ++-
 .../python-modules/fasteners/default.nix           |   2 +
 .../python-modules/fastparquet/default.nix         |   4 +-
 .../python-modules/findpython/default.nix          |   4 +-
 pkgs/development/python-modules/fitbit/default.nix |  40 +-
 .../python-modules/flake8-blind-except/default.nix |   5 +
 pkgs/development/python-modules/flake8/default.nix |   2 +-
 .../python-modules/flask-login/default.nix         |   4 +-
 .../python-modules/flask-paranoid/default.nix      |   4 +-
 .../python-modules/flask-restful/default.nix       |   4 +
 .../flask-restful/werkzeug-2.1.0-compat.patch      | 114 ++++
 .../python-modules/flask-restx/default.nix         |  15 +
 .../python-modules/flask-security-too/default.nix  | 114 +++-
 .../python-modules/flask-socketio/default.nix      |   8 +-
 .../python-modules/flask-talisman/default.nix      |   4 +-
 pkgs/development/python-modules/flask/default.nix  |  14 +-
 pkgs/development/python-modules/flax/default.nix   |   5 +-
 .../python-modules/flickrapi/default.nix           |  69 ++-
 .../python-modules/flit-core/default.nix           |   4 +-
 pkgs/development/python-modules/flit/default.nix   |  21 +-
 pkgs/development/python-modules/fn/default.nix     |  30 +-
 .../python-modules/fontparts/default.nix           |   8 +-
 .../python-modules/fonttools/default.nix           |   7 +-
 .../python-modules/freezegun/default.nix           |  31 +-
 .../python-modules/funcparserlib/default.nix       |   2 +
 .../python-modules/geoalchemy2/default.nix         |   4 +-
 pkgs/development/python-modules/gipc/default.nix   |   2 -
 .../python-modules/glean-parser/default.nix        |  11 +-
 .../python-modules/google-api-core/default.nix     |   4 +-
 .../google-api-python-client/default.nix           |   4 +-
 .../google-auth-oauthlib/default.nix               |  16 +-
 .../google-cloud-bigquery/default.nix              |   4 +-
 .../google-cloud-testutils/default.nix             |   4 +-
 .../python-modules/google-nest-sdm/default.nix     |   4 +-
 .../python-modules/graphql-relay/default.nix       |  23 +-
 pkgs/development/python-modules/grip/default.nix   |  13 +-
 .../python-modules/grpcio-status/default.nix       |   4 +-
 .../python-modules/grpcio-tools/default.nix        |   4 +-
 pkgs/development/python-modules/h11/default.nix    |   3 +
 pkgs/development/python-modules/hidapi/default.nix |   4 +-
 .../python-modules/homeconnect/default.nix         |  15 +-
 pkgs/development/python-modules/hpack/default.nix  |  25 +-
 .../development/python-modules/httpbin/default.nix |  10 +
 .../python-modules/httplib2/default.nix            |   6 +-
 .../python-modules/hyperframe/default.nix          |   5 +-
 .../python-modules/hypothesis/default.nix          |   4 +-
 .../python-modules/hypothesmith/default.nix        |  32 +-
 .../python-modules/hypothesmith/remove-black.patch |  88 +++
 .../python-modules/ibis-framework/default.nix      |   1 -
 .../development/python-modules/img2pdf/default.nix |  14 +-
 .../python-modules/importlib-metadata/default.nix  |   4 +-
 .../python-modules/importlib-resources/default.nix |   5 +-
 .../python-modules/installer/default.nix           |   4 +-
 .../python-modules/intbitset/default.nix           |   4 +-
 .../intensity-normalization/default.nix            |   4 +-
 .../python-modules/ipykernel/default.nix           |  16 +-
 .../development/python-modules/ipython/default.nix |   8 +-
 .../python-modules/ipywidgets/default.nix          |   4 +-
 pkgs/development/python-modules/islpy/default.nix  |   4 +-
 pkgs/development/python-modules/iterm2/default.nix |   4 +-
 .../python-modules/itsdangerous/default.nix        |   4 +-
 .../python-modules/jarowinkler/default.nix         |  54 ++
 pkgs/development/python-modules/jax/default.nix    |  18 +-
 pkgs/development/python-modules/jaxlib/default.nix |   2 +
 pkgs/development/python-modules/jinja2/default.nix |   7 +-
 pkgs/development/python-modules/jira/default.nix   |  31 +-
 .../python-modules/jmespath/default.nix            |   4 +-
 pkgs/development/python-modules/josepy/default.nix |   4 +-
 .../json-schema-for-humans/default.nix             |  11 +-
 .../python-modules/junos-eznc/default.nix          |  64 +-
 .../python-modules/jupyter-client/default.nix      |   4 +-
 .../python-modules/jupyter-packaging/default.nix   |   4 +-
 .../python-modules/jupyter_console/default.nix     |   4 +-
 .../python-modules/jupyterlab-git/default.nix      |   4 +-
 .../python-modules/jupyterlab_server/default.nix   |   4 +-
 .../python-modules/kivy-garden/default.nix         |  20 +-
 .../python-modules/kiwisolver/default.nix          |  30 +-
 .../python-modules/kubernetes/default.nix          |   7 +-
 .../python-modules/levenshtein/default.nix         |  20 +-
 pkgs/development/python-modules/limits/default.nix |   4 +-
 .../python-modules/lmnotify/default.nix            |  23 +-
 .../python-modules/markupsafe/default.nix          |   8 +-
 .../marshmallow-sqlalchemy/default.nix             |   4 +-
 .../development/python-modules/matchpy/default.nix |  52 +-
 pkgs/development/python-modules/mccabe/default.nix |  20 +-
 .../python-modules/metakernel/default.nix          |   4 +-
 .../python-modules/mezzanine/default.nix           |  68 ++-
 .../python-modules/mitmproxy/default.nix           |   1 +
 .../python-modules/mkdocs-material/default.nix     |   4 +-
 pkgs/development/python-modules/mkdocs/default.nix |  13 +-
 pkgs/development/python-modules/mock/default.nix   |  26 +-
 .../development/python-modules/modeled/default.nix |   4 +-
 .../python-modules/moretools/default.nix           |   4 +-
 pkgs/development/python-modules/moto/default.nix   |  22 +-
 pkgs/development/python-modules/msrest/default.nix |  69 ++-
 .../python-modules/mwclient/default.nix            |  38 +-
 .../development/python-modules/mwoauth/default.nix |  23 +-
 .../python-modules/myfitnesspal/default.nix        |   2 +
 .../python-modules/mypy-boto3-s3/default.nix       |   4 +-
 .../python-modules/nbclient/default.nix            |   4 +-
 .../python-modules/nbconvert/default.nix           |   7 +-
 .../python-modules/nbformat/default.nix            |   4 +-
 .../python-modules/ncclient/default.nix            |   4 +-
 .../python-modules/ndg-httpsclient/default.nix     |   9 +-
 .../development/python-modules/netmiko/default.nix |   4 +-
 .../python-modules/networkx/default.nix            |   4 +-
 .../python-modules/nipype/neurdflib.nix            |   4 +-
 .../python-modules/nocasedict/default.nix          |  13 +-
 .../python-modules/notebook/default.nix            |   4 +-
 .../python-modules/numpydoc/default.nix            |  45 +-
 .../development/python-modules/nunavut/default.nix |   6 +-
 pkgs/development/python-modules/oci/default.nix    |   4 +-
 pkgs/development/python-modules/ondilo/default.nix |  13 +-
 .../python-modules/opentracing/default.nix         |  19 +-
 pkgs/development/python-modules/optax/default.nix  |   1 -
 pkgs/development/python-modules/ovh/default.nix    |  35 +-
 .../python-modules/packaging/default.nix           |  54 +-
 .../python-modules/paho-mqtt/default.nix           |  10 +-
 pkgs/development/python-modules/pandas/default.nix |  12 +-
 .../python-modules/parsimonious/default.nix        |  25 +-
 .../development/python-modules/path.py/default.nix |  50 --
 pkgs/development/python-modules/path/default.nix   |  51 ++
 .../python-modules/pdm-pep517/default.nix          |   4 +-
 pkgs/development/python-modules/pep257/default.nix |  23 -
 pkgs/development/python-modules/phik/default.nix   |   4 +-
 .../python-modules/phonemizer/default.nix          |   4 +-
 .../python-modules/pickleshare/default.nix         |   4 +-
 .../development/python-modules/pikepdf/default.nix |   4 +
 pkgs/development/python-modules/pillow/default.nix |  18 +-
 pkgs/development/python-modules/pip/default.nix    |   4 +-
 .../python-modules/pleroma-bot/default.nix         |  29 +-
 .../python-modules/prettytable/default.nix         |   4 +-
 .../python-modules/prompt-toolkit/default.nix      |   2 +
 .../python-modules/psautohint/default.nix          |  16 +-
 pkgs/development/python-modules/purl/default.nix   |  17 +-
 .../python-modules/py-radix/default.nix            |   3 +
 pkgs/development/python-modules/py4j/default.nix   |   4 +-
 pkgs/development/python-modules/pyasn1/default.nix |  10 +-
 pkgs/development/python-modules/pyatmo/default.nix |   4 +-
 pkgs/development/python-modules/pybids/default.nix |   4 +-
 .../python-modules/pybotvac/default.nix            |  14 +-
 .../python-modules/pycountry/default.nix           |   4 +-
 pkgs/development/python-modules/pycurl/default.nix |  10 -
 .../python-modules/pyfakefs/default.nix            |   4 +-
 pkgs/development/python-modules/pygit2/default.nix |   4 +-
 .../python-modules/pykerberos/default.nix          |   4 +-
 pkgs/development/python-modules/pylint/default.nix |  28 +-
 pkgs/development/python-modules/pymc3/default.nix  |   4 +-
 .../python-modules/pymemcache/default.nix          |   6 +
 pkgs/development/python-modules/pymfy/default.nix  |  13 +-
 .../development/python-modules/pympler/default.nix |  13 +-
 .../development/python-modules/pynello/default.nix |  10 +-
 .../python-modules/pyopenssl/default.nix           |   8 +-
 .../python-modules/pyparsing/default.nix           |  10 +-
 pkgs/development/python-modules/pyrad/default.nix  |  47 +-
 .../python-modules/pyramid_jinja2/default.nix      |   4 +-
 .../python-modules/pyro-ppl/default.nix            |   4 +-
 .../development/python-modules/pyslurm/default.nix |  19 +-
 .../python-modules/pysmappee/default.nix           |  16 +-
 pkgs/development/python-modules/pysnow/default.nix |  58 +-
 .../development/python-modules/pyspark/default.nix |   2 +-
 .../python-modules/pytest-asyncio/default.nix      |  12 +-
 .../python-modules/pytest-check/default.nix        |   4 +-
 .../python-modules/pytest-describe/default.nix     |  40 ++
 .../python-modules/pytest-httpbin/default.nix      |   6 +
 .../python-modules/pytest-metadata/default.nix     |  15 +-
 .../python-modules/pytest-mock/default.nix         |   9 +
 .../python-modules/pytest-pep257/default.nix       |  20 -
 .../python-modules/pytest-shutil/default.nix       |  11 +-
 .../python-modules/pytest-testmon/default.nix      |   4 +-
 .../python-modules/pytest-xdist/default.nix        |  11 +-
 .../python-modules/pytest-xdist/setup-hook.sh      |  18 +
 pkgs/development/python-modules/pytest/default.nix |  51 +-
 .../python-modules/python-dotenv/default.nix       |   4 +-
 .../python-modules/python-google-nest/default.nix  |   6 +-
 .../python-modules/python-logstash/default.nix     |   4 +-
 .../python-modules/python-registry/default.nix     |   2 +
 .../python-modules/python-twitter/default.nix      |  39 +-
 .../python-modules/python-vagrant/default.nix      |   4 +-
 .../python-modules/python3-saml/default.nix        |   5 +
 .../development/python-modules/pytools/default.nix |   4 +-
 pkgs/development/python-modules/pytorch/bin.nix    |   1 +
 pkgs/development/python-modules/pytz/default.nix   |   4 +-
 .../python-modules/pyvicare/default.nix            |   8 +-
 .../python-modules/pywavelets/default.nix          |   4 +-
 .../python-modules/qtconsole/default.nix           |   4 +-
 .../python-modules/rapidfuzz-capi/default.nix      |  28 +
 .../python-modules/rapidfuzz/default.nix           |  34 +-
 .../readthedocs-sphinx-ext/default.nix             |   4 +-
 pkgs/development/python-modules/redis/default.nix  |  31 +-
 .../python-modules/reedsolo/default.nix            |  21 +-
 .../python-modules/reportlab/default.nix           |  10 +-
 .../python-modules/requests/default.nix            |  20 +-
 .../python-modules/responses/default.nix           |  12 +-
 pkgs/development/python-modules/rich/default.nix   |   9 +-
 .../python-modules/ring-doorbell/default.nix       |  18 +-
 .../robotframework-tools/default.nix               |   4 +-
 .../python-modules/sagemaker/default.nix           |   4 +-
 pkgs/development/python-modules/sanic/default.nix  |   2 +
 .../python-modules/scikit-build/default.nix        |  56 +-
 .../python-modules/scikit-learn/default.nix        |   2 -
 .../python-modules/service_identity/default.nix    |   2 +
 .../python-modules/setuptools-rust/default.nix     |   6 +-
 .../python-modules/setuptools/default.nix          |   4 +-
 .../setuptools/setuptools-distutils-C++.patch      |  72 +--
 .../development/python-modules/shapely/default.nix |   4 +-
 .../python-modules/shapely/library-paths.patch     |  72 ++-
 .../python-modules/social-auth-core/default.nix    |   4 +-
 .../python-modules/sortedcontainers/default.nix    |  47 +-
 .../python-modules/spacy-transformers/default.nix  |   4 +-
 pkgs/development/python-modules/spacy/default.nix  |   4 +-
 .../python-modules/sparqlwrapper/default.nix       |  17 +-
 .../python-modules/sphinx-jinja/default.nix        |  21 +-
 pkgs/development/python-modules/sphinx/default.nix |  13 +-
 .../python-modules/sphinx_rtd_theme/default.nix    |   2 +-
 .../python-modules/spyder-kernels/default.nix      |   4 +-
 pkgs/development/python-modules/spyder/default.nix |   4 +-
 .../sqlalchemy-imageattach/default.nix             |  42 --
 .../python-modules/sqlalchemy/default.nix          |   4 +-
 .../python-modules/starlette/default.nix           |   4 +-
 .../python-modules/svg-path/default.nix            |  24 +-
 .../python-modules/symengine/default.nix           |  15 +-
 pkgs/development/python-modules/sympy/default.nix  |   4 +-
 .../python-modules/tellduslive/default.nix         |  13 +-
 .../python-modules/tensorflow-datasets/default.nix |   1 -
 .../python-modules/terminado/default.nix           |   4 +-
 .../python-modules/tifffile/default.nix            |   4 +-
 .../development/python-modules/tomlkit/default.nix |   4 +-
 .../python-modules/torchinfo/default.nix           |   4 +-
 .../development/python-modules/tornado/default.nix |  22 +-
 .../python-modules/tpm2-pytss/default.nix          |   4 +-
 pkgs/development/python-modules/tqdm/default.nix   |   4 +-
 .../development/python-modules/trytond/default.nix |   9 +
 pkgs/development/python-modules/tweepy/default.nix |  12 +-
 pkgs/development/python-modules/twine/default.nix  |  24 +-
 .../development/python-modules/twisted/default.nix |  17 +-
 .../python-modules/twitterapi/default.nix          |   6 +-
 pkgs/development/python-modules/typer/default.nix  |  19 +-
 pkgs/development/python-modules/tzdata/default.nix |   4 +-
 pkgs/development/python-modules/uamqp/default.nix  |   4 +-
 pkgs/development/python-modules/ufo2ft/default.nix |   4 +-
 .../development/python-modules/urllib3/default.nix |   4 +-
 pkgs/development/python-modules/ush/default.nix    |  12 +-
 .../python-modules/validators/default.nix          |   2 +
 .../python-modules/validictory/default.nix         |   3 +
 .../python-modules/variants/default.nix            |   3 +-
 .../python-modules/vdirsyncer/default.nix          |  21 +-
 .../python-modules/virtualenv/default.nix          |   4 +-
 .../python-modules/waitress/default.nix            |   2 +-
 pkgs/development/python-modules/wasm/default.nix   |   3 +
 .../python-modules/watchdog/default.nix            |   4 +-
 .../python-modules/watchgod/default.nix            |   9 +-
 .../python-modules/websocket-client/default.nix    |   6 +-
 .../python-modules/weconnect-mqtt/default.nix      |   4 +-
 .../python-modules/weconnect/default.nix           |   4 +-
 .../python-modules/werkzeug/default.nix            |  13 +-
 .../python-modules/widgetsnbextension/default.nix  |   4 +-
 .../development/python-modules/wsproto/default.nix |  19 +-
 pkgs/development/python-modules/wxPython/4.1.nix   |   2 +
 .../development/python-modules/yamlfix/default.nix |   5 -
 pkgs/development/python-modules/yanc/default.nix   |  10 +-
 pkgs/development/python-modules/zarr/default.nix   |   4 +-
 pkgs/development/python-modules/zetup/default.nix  |   4 +-
 pkgs/development/python-modules/zodb/default.nix   |   4 +-
 .../python-modules/zope_testing/default.nix        |   4 +-
 pkgs/development/python-modules/zopfli/default.nix |  11 +-
 .../cryptography-vectors/default.nix               |  23 -
 .../python2-modules/cryptography/default.nix       |  15 +-
 .../python2-modules/cryptography/vectors.nix       |  24 +
 .../ruby-modules/bundler-app/default.nix           |   2 +-
 pkgs/development/tools/analysis/coan/default.nix   |   4 +
 pkgs/development/tools/analysis/codeql/default.nix |   1 +
 .../build-managers/bazel/bazel_0_29/default.nix    |   3 +-
 .../tools/build-managers/bazel/bazel_1/default.nix |   3 +-
 .../tools/build-managers/bazel/bazel_3/default.nix |   3 +-
 .../tools/build-managers/bazel/bazel_4/default.nix |   3 +-
 .../tools/build-managers/bloop/default.nix         |   3 +-
 .../tools/build-managers/wafHook/setup-hook.sh     |   4 +
 pkgs/development/tools/ccloud-cli/default.nix      |   2 +-
 pkgs/development/tools/confluent-cli/default.nix   |   2 +-
 .../development/tools/database/litecli/default.nix |   4 +-
 .../tools/database/sqldeveloper/default.nix        |   2 +-
 ...vert-libtool.m4-fix-nm-BSD-flag-detection.patch | 137 +++++
 ...l.m4-update-macos-version-detection-block.patch |  47 ++
 .../tools/misc/binutils/CVE-2020-35448.patch       |  77 ---
 .../tools/misc/binutils/CVE-2021-3487.patch        |  73 ---
 .../tools/misc/binutils/CVE-2021-45078.patch       | 239 --------
 ...f-Dont-read-non-existing-secondary-relocs.patch |  26 -
 .../binutils/build-components-separately.patch     |  67 +--
 pkgs/development/tools/misc/binutils/default.nix   | 221 +++----
 .../misc/binutils/disambiguate-arm-targets.patch   |  23 -
 .../gold-Update-GNU_PROPERTY_X86_XXX-macros.patch  | 292 ---------
 .../tools/misc/libtool/libtool2-macos11.patch      |  32 -
 pkgs/development/tools/misc/libtool/libtool2.nix   |  26 +-
 pkgs/development/tools/mold/default.nix            |   4 +-
 pkgs/development/tools/rust/cbindgen/default.nix   |   6 +-
 pkgs/development/web/nodejs/nodejs.nix             |  21 +-
 pkgs/games/airstrike/default.nix                   |   2 +-
 pkgs/games/assaultcube/default.nix                 |   4 +-
 pkgs/games/deliantra/arch.nix                      |  25 +
 pkgs/games/deliantra/data.nix                      |  21 +
 pkgs/games/deliantra/default.nix                   |  10 -
 pkgs/games/deliantra/deliantra-arch.nix            |  25 -
 pkgs/games/deliantra/deliantra-data.nix            |  21 -
 pkgs/games/deliantra/deliantra-maps.nix            |  25 -
 pkgs/games/deliantra/deliantra-server.nix          |  48 --
 pkgs/games/deliantra/maps.nix                      |  25 +
 pkgs/games/deliantra/server.nix                    |  52 ++
 pkgs/games/domination/default.nix                  |   4 +-
 pkgs/games/enigma/default.nix                      |  29 +-
 pkgs/games/flare/default.nix                       |   2 +-
 pkgs/games/frogatto/default.nix                    |   2 +-
 pkgs/games/iortcw/default.nix                      |   2 +-
 pkgs/games/iortcw/sp.nix                           |   2 +-
 pkgs/games/minecraft/default.nix                   |   2 +-
 pkgs/games/mudlet/default.nix                      |   2 +-
 pkgs/games/nanosaur/default.nix                    |   2 +-
 pkgs/games/openjk/default.nix                      |   8 +-
 pkgs/games/openmw/tes3mp.nix                       |   4 +-
 pkgs/games/openra/common.nix                       |   2 +-
 pkgs/games/otto-matic/default.nix                  |   2 +-
 pkgs/games/runelite/default.nix                    |   2 +-
 pkgs/games/sauerbraten/default.nix                 |   4 +-
 pkgs/games/starsector/default.nix                  |   3 +-
 pkgs/games/tome4/default.nix                       |   2 +-
 pkgs/games/ut2004/wrapper.nix                      |   2 +-
 pkgs/games/warsow/default.nix                      |   2 +-
 pkgs/games/zod/default.nix                         |   4 +-
 pkgs/misc/jackaudio/default.nix                    |   2 -
 pkgs/misc/rich-cli/default.nix                     |  32 +
 pkgs/os-specific/linux/bluez/default.nix           |   4 +-
 pkgs/os-specific/linux/displaylink/default.nix     |   2 +-
 pkgs/os-specific/linux/keyutils/default.nix        |  14 +-
 pkgs/os-specific/linux/libcap/default.nix          |   1 +
 pkgs/os-specific/linux/lvm2/common.nix             |  21 +-
 .../os-specific/linux/lvm2/fix-blkdeactivate.patch |  51 ++
 pkgs/os-specific/linux/sgx/psw/default.nix         |   2 +-
 pkgs/os-specific/linux/sgx/samples/default.nix     |   4 +-
 pkgs/servers/home-assistant/cli.nix                |  20 +-
 pkgs/servers/home-assistant/default.nix            |   2 -
 pkgs/servers/home-assistant/tests.nix              |   4 +
 pkgs/servers/hqplayerd/default.nix                 |   1 +
 pkgs/servers/mail/archiveopteryx/default.nix       |  10 +-
 pkgs/servers/mail/mailman/default.nix              |   8 +-
 pkgs/servers/ombi/default.nix                      |   5 +-
 pkgs/servers/plik/default.nix                      |   2 +-
 .../add-option-for-installation-sysconfdir.patch   |  36 +-
 pkgs/servers/pulseaudio/correct-ldflags.patch      |  13 -
 pkgs/servers/pulseaudio/default.nix                |  95 ++-
 pkgs/servers/pulseaudio/pali.nix                   | 219 -------
 pkgs/servers/roon-bridge/default.nix               |   2 +-
 pkgs/servers/roon-server/default.nix               |   4 +-
 pkgs/servers/search/elasticsearch/6.x.nix          |   5 +-
 pkgs/servers/web-apps/discourse/default.nix        |   2 +-
 pkgs/servers/web-apps/virtlyst/default.nix         |   7 +-
 pkgs/servers/x11/xorg/default.nix                  |  22 +-
 .../x11/xorg/generate-expr-from-tarballs.pl        |   6 +-
 pkgs/servers/x11/xorg/overrides.nix                |  31 +-
 pkgs/servers/x11/xorg/tarballs.list                |   6 +-
 pkgs/shells/powershell/default.nix                 |   3 +-
 pkgs/stdenv/generic/setup.sh                       |  26 +
 pkgs/stdenv/linux/default.nix                      |   2 +-
 pkgs/stdenv/linux/make-bootstrap-tools.nix         |   5 +-
 pkgs/tools/X11/nx-libs/default.nix                 |   6 +
 pkgs/tools/admin/awscli/default.nix                |   4 +-
 pkgs/tools/admin/oci-cli/default.nix               |  12 +-
 pkgs/tools/admin/pgadmin/default.nix               |   3 +-
 pkgs/tools/audio/beets/default.nix                 |   2 +-
 pkgs/tools/backup/borgbackup/default.nix           |   3 +-
 pkgs/tools/backup/gphotos-sync/default.nix         |  16 +-
 pkgs/tools/backup/grab-site/default.nix            |   3 +-
 pkgs/tools/cd-dvd/ventoy-bin/default.nix           |   4 +-
 pkgs/tools/filesystems/e2fsprogs/default.nix       |  11 +-
 pkgs/tools/games/ajour/default.nix                 |   4 +-
 .../tools/graphics/snapdragon-profiler/default.nix |   2 +-
 pkgs/tools/graphics/structure-synth/default.nix    |   2 +-
 pkgs/tools/misc/archi/default.nix                  |   3 +-
 pkgs/tools/misc/coreutils/default.nix              |   1 +
 pkgs/tools/misc/findutils/default.nix              |  19 +-
 pkgs/tools/misc/lookatme/default.nix               |  12 +-
 pkgs/tools/misc/man-db/default.nix                 |   4 +-
 pkgs/tools/misc/pre-commit/default.nix             |   1 -
 pkgs/tools/networking/cjdns/default.nix            |  10 +-
 pkgs/tools/package-management/pdm/default.nix      |   5 +-
 pkgs/tools/security/fwbuilder/default.nix          |   4 +
 pkgs/tools/virtualization/awsebcli/default.nix     |   1 +
 pkgs/top-level/aliases.nix                         |   3 +
 pkgs/top-level/all-packages.nix                    | 125 ++--
 pkgs/top-level/python-aliases.nix                  |  10 +-
 pkgs/top-level/python-packages.nix                 |  36 +-
 pkgs/top-level/python2-packages.nix                |   2 -
 640 files changed, 5980 insertions(+), 4872 deletions(-)
 delete mode 100644 pkgs/applications/audio/libopenmpt/default.nix
 delete mode 100644 pkgs/applications/audio/pulseaudio-modules-bt/default.nix
 delete mode 100644 pkgs/applications/audio/pulseaudio-modules-bt/fix-install-path.patch
 delete mode 100644 pkgs/development/compilers/rust/1_59.nix
 create mode 100644 pkgs/development/compilers/rust/1_60.nix
 delete mode 100644 pkgs/development/compilers/rust/rustfmt-fix-self-tests.patch
 create mode 100644 pkgs/development/interpreters/ruby/rbinstall-new-rubygems-compat.patch
 create mode 100644 pkgs/development/libraries/audio/libopenmpt/default.nix
 create mode 100755 pkgs/development/libraries/audio/libopenmpt/update.sh
 create mode 100644 pkgs/development/libraries/glib/elementary-terminal-support.patch
 create mode 100644 pkgs/development/libraries/glib/gnome-console-support.patch
 delete mode 100644 pkgs/development/libraries/libnfnetlink/Use-stdlib-uint-instead-of-u_int.patch
 create mode 100644 pkgs/development/libraries/libotr/fix-regtest-client.patch
 delete mode 100644 pkgs/development/libraries/nss/default.nix
 create mode 100644 pkgs/development/libraries/nss/esr.nix
 create mode 100644 pkgs/development/libraries/nss/generic.nix
 create mode 100644 pkgs/development/libraries/nss/latest.nix
 delete mode 100644 pkgs/development/libraries/qt-5/5.14/qtwayland-libdrm-build.patch
 create mode 100644 pkgs/development/libraries/zlib/CVE-2018-25032-1.patch
 create mode 100644 pkgs/development/libraries/zlib/CVE-2018-25032-2.patch
 create mode 100644 pkgs/development/python-modules/ephemeral-port-reserve/default.nix
 create mode 100644 pkgs/development/python-modules/flask-restful/werkzeug-2.1.0-compat.patch
 create mode 100644 pkgs/development/python-modules/hypothesmith/remove-black.patch
 create mode 100644 pkgs/development/python-modules/jarowinkler/default.nix
 delete mode 100644 pkgs/development/python-modules/path.py/default.nix
 create mode 100644 pkgs/development/python-modules/path/default.nix
 delete mode 100644 pkgs/development/python-modules/pep257/default.nix
 create mode 100644 pkgs/development/python-modules/pytest-describe/default.nix
 delete mode 100644 pkgs/development/python-modules/pytest-pep257/default.nix
 create mode 100644 pkgs/development/python-modules/pytest-xdist/setup-hook.sh
 create mode 100644 pkgs/development/python-modules/rapidfuzz-capi/default.nix
 delete mode 100644 pkgs/development/python-modules/sqlalchemy-imageattach/default.nix
 delete mode 100644 pkgs/development/python2-modules/cryptography-vectors/default.nix
 create mode 100644 pkgs/development/python2-modules/cryptography/vectors.nix
 create mode 100644 pkgs/development/tools/misc/binutils/0001-Revert-libtool.m4-fix-nm-BSD-flag-detection.patch
 create mode 100644 pkgs/development/tools/misc/binutils/0001-libtool.m4-update-macos-version-detection-block.patch
 delete mode 100644 pkgs/development/tools/misc/binutils/CVE-2020-35448.patch
 delete mode 100644 pkgs/development/tools/misc/binutils/CVE-2021-3487.patch
 delete mode 100644 pkgs/development/tools/misc/binutils/CVE-2021-45078.patch
 delete mode 100644 pkgs/development/tools/misc/binutils/bfd-elf-Dont-read-non-existing-secondary-relocs.patch
 delete mode 100644 pkgs/development/tools/misc/binutils/disambiguate-arm-targets.patch
 delete mode 100644 pkgs/development/tools/misc/binutils/gold-Update-GNU_PROPERTY_X86_XXX-macros.patch
 delete mode 100644 pkgs/development/tools/misc/libtool/libtool2-macos11.patch
 create mode 100644 pkgs/games/deliantra/arch.nix
 create mode 100644 pkgs/games/deliantra/data.nix
 delete mode 100644 pkgs/games/deliantra/default.nix
 delete mode 100644 pkgs/games/deliantra/deliantra-arch.nix
 delete mode 100644 pkgs/games/deliantra/deliantra-data.nix
 delete mode 100644 pkgs/games/deliantra/deliantra-maps.nix
 delete mode 100644 pkgs/games/deliantra/deliantra-server.nix
 create mode 100644 pkgs/games/deliantra/maps.nix
 create mode 100644 pkgs/games/deliantra/server.nix
 create mode 100644 pkgs/misc/rich-cli/default.nix
 create mode 100644 pkgs/os-specific/linux/lvm2/fix-blkdeactivate.patch
 delete mode 100644 pkgs/servers/pulseaudio/correct-ldflags.patch
 delete mode 100644 pkgs/servers/pulseaudio/pali.nix
binäre Suche fand ersten schlechten Commit
```
</details>

Looking at the [test-requirements](https://github.com/getsentry/sentry-python/blob/master/test-requirements.txt) there is a requirement for werkzeug < 2.1.0.
When applied, the test mostly pass. The only failing test is:
```
=================================== FAILURES ===================================
___________________________ test_cli_commands_raise ____________________________
tests/integrations/flask/test_flask.py:503: in test_cli_commands_raise
    app.cli.main(
/nix/store/jx97i20a84abbr6hsalwxvcr9m00sphh-python3.9-click-8.1.2/lib/python3.9/site-packages/click/core.py:1055: in main
    rv = self.invoke(ctx)
/nix/store/jx97i20a84abbr6hsalwxvcr9m00sphh-python3.9-click-8.1.2/lib/python3.9/site-packages/click/core.py:1657: in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
/nix/store/jx97i20a84abbr6hsalwxvcr9m00sphh-python3.9-click-8.1.2/lib/python3.9/site-packages/click/core.py:1404: in invoke
    return ctx.invoke(self.callback, **ctx.params)
/nix/store/jx97i20a84abbr6hsalwxvcr9m00sphh-python3.9-click-8.1.2/lib/python3.9/site-packages/click/core.py:760: in invoke
    return __callback(*args, **kwargs)
/nix/store/jx97i20a84abbr6hsalwxvcr9m00sphh-python3.9-click-8.1.2/lib/python3.9/site-packages/click/decorators.py:26: in new_func
    return f(get_current_context(), *args, **kwargs)
/nix/store/hpscxkmgv6lvvygm480nkd32iyhwwss8-python3.9-Flask-2.1.1/lib/python3.9/site-packages/flask/cli.py:426: in decorator
    with __ctx.ensure_object(ScriptInfo).load_app().app_context():
/nix/store/hpscxkmgv6lvvygm480nkd32iyhwwss8-python3.9-Flask-2.1.1/lib/python3.9/site-packages/flask/cli.py:382: in load_app
    app = self.create_app()
E   TypeError: <lambda>() missing 1 required positional argument: '_'
```

There is also a requirement for pytest < 7. Since this packaged builds with the latest nixos/unstable, which includes pytest 7.0.1 I am now testing with that version.

--- a/pkgs/development/python-modules/sentry-sdk/default.nix
+++ b/pkgs/development/python-modules/sentry-sdk/default.nix
@@ -137,10 +137,10 @@ buildPythonPackage rec {
     "tests/integrations/pyramid/test_pyramid.py"
   ]
   # test crashes on aarch64
-  ++ (if stdenv.buildPlatform != "x86_64-linux" then [
+  ++ lib.optionals (stdenv.buildPlatform != "x86_64-linux") [
     "tests/test_transport.py"
     "tests/integrations/threading/test_threading.py"
-  ] else [ ]);
+  ];
 
   pythonImportsCheck = [
     "sentry_sdk"

--- a/pkgs/development/python-modules/sentry-sdk/default.nix
+++ b/pkgs/development/python-modules/sentry-sdk/default.nix
@@ -112,6 +112,8 @@ buildPythonPackage rec {
     "test_auto_session_tracking_with_aggregates"
     # Network requests to public web
     "test_crumb_capture"
+    # TypeError: cannot unpack non-iterable TestResponse object
+    "test_rpc_error_page"
   ];
 
   disabledTestPaths = [
@@ -128,7 +130,17 @@ buildPythonPackage rec {
     "tests/integrations/rq/"
     # broken since pytest 7.0.1; AssertionError: previous item was not torn down properly
     "tests/utils/test_contextvars.py"
-  ];
+    # broken since Flask and Werkzeug update to 2.1.0 (different error messages)
+    "tests/integrations/flask/test_flask.py"
+    "tests/integrations/bottle/test_bottle.py"
+    "tests/integrations/django/test_basic.py"
+    "tests/integrations/pyramid/test_pyramid.py"
+  ]
+  # test crashes on aarch64
+  ++ (if stdenv.buildPlatform != "x86_64-linux" then [
+    "tests/test_transport.py"
+    "tests/integrations/threading/test_threading.py"
+  ] else [ ]);
 
   pythonImportsCheck = [
     "sentry_sdk"

